test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
+    {
+        ProjectTestExtension _webProject;
+        ProjectItemTestExtension _libManConfig;
+        const string _projectName = @"TestProjectCore20";
+        const string _libman = "libman.json";
+        private string _initialLibmanFileContent;
+        private string _pathToLibmanFile;
+
+        [TestInitialize]
```

Comment:
>TestInitialize [](start = 9, length = 14)

I wonder if it's better to override DoHostTestInitialize as it appears this code has an ordering dependency with it.

Reply:
Updated in iteration 2.

---
In reply to: [205457432](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205457432) [](ancestors = 205457432)